src/main/java/com/deque/axe/AXE.java

```diff
@@ -366,7 +366,8 @@ private JSONObject execute(final String command, final Object... args) {
 			this.driver.manage().timeouts().setScriptTimeout(this.timeout, TimeUnit.SECONDS);
 
 			Object response = ((JavascriptExecutor) this.driver).executeAsyncScript(command, args);
-			JSONObject result = new JSONObject((Map) response);
+            JSONObject result = new JSONObject((Map)response);
```

Comment:
This change is unnecessary. Please revert.

```suggestion
			JSONObject result = new JSONObject((Map) response);
```

Reply:
Fixed